packages/walletconnect/src/index.ts

```diff
@@ -10,61 +9,44 @@ export const URI_AVAILABLE = 'URI_AVAILABLE'
 
 type MockWalletConnectProvider = WalletConnectProvider & EventEmitter
 
-function parseChainId(chainId: string | number) {
-  return typeof chainId === 'string' ? Number.parseInt(chainId) : chainId
-}
-
-type WalletConnectOptions = Omit<IWCEthRpcConnectionOptions, 'rpc' | 'infuraId' | 'chainId'> & {
-  rpc: { [chainId: number]: string | string[] }
+// @todo document why `rpcMap` type is changed
+type WalletConnectOptions = Omit<Parameters<typeof WalletConnectProvider.init>[0], 'rpcMap'> & {
+  rpcMap?: { [chainId: number]: string | string[] }
 }
 
 /**
  * @param options - Options to pass to `@walletconnect/ethereum-provider`
- * @param defaultChainId - The chainId to connect to in activate if one is not provided.
  * @param timeout - Timeout, in milliseconds, after which to treat network calls to urls as failed when selecting
  * online urls.
  * @param onError - Handler to report errors thrown from eventListeners.
  */
 export interface WalletConnectConstructorArgs {
   actions: Actions
   options: WalletConnectOptions
-  defaultChainId?: number
   timeout?: number
   onError?: (error: Error) => void
 }
 
-/**
- * @param desiredChainId - The desired chainId to connect to.
```

Comment:
this code wasn't referred anywhere in the codebase on main, so I got rid of it - anyone has additional context?